test/CMakeLists.txt

```diff
@@ -217,7 +217,6 @@ set( LONG_TESTS
     pooling3d.cpp
     soft_max.cpp
     lrn_test.cpp
-    reduce_test.cpp
```

Comment:
Please revert this. It tests reduce in accordance with the datatype of the stage (as per to MIOPEN_TEST_HALF/IN8/BFLOAT16 or default FP32).

Reply:
FYI this block of code ensures that reduce_test is used with FLOAT and HALF on gfx906/900 and gfx908:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/96d7086fa0113e2109de44f870dbae1c340b48fc/test/CMakeLists.txt#L93-L105

Reply:
> Please revert this. It tests reduce in accordance with the datatype of the stage (as per to MIOPEN_TEST_HALF/IN8/BFLOAT16 or default FP32).

Please ignora the above. Currently LONG_TESTS is not used at all. It's an old leftover that should be eventually removed.